2-0-data-structures-and-algorithms/src/test/java/com/bobocode/array_list/ArrayListTest.java

```diff
@@ -94,9 +109,16 @@ public void getLastOfEmptyList() {
     public void listWithSpecificCapacity() {
         arrayList = new ArrayList<>(8);
 
-        arrayList.add(10);
-        arrayList.add(15);
-        arrayList.add(20);
+        try {
```

Comment:
@erneman1 this logic is redundant, we just need to **verify that array capacity equals the one we passed to the constructor**. So please remove this logic as well as assertions, **verify that the inner array length is correct.**